lib/validates_identity/usage.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class ValidatesIdentity
+  class Usage
+    attr_reader :identity_type, :only
+
+    def initialize(identity_type, only)
+      @identity_type = identity_type
+      @only = only
+    end
+
+    def valid?
+      return false if identity_type.blank? && only.blank?
+      return false if identity_type.blank? && %i[person legal].include?(only)
+
+      identity_type.blank? || [:person, :legal, nil].include?(only)
+    end
```

Comment:
What if we improve this code a bit?

Something like

```ruby
if identity_type.present? && [:person, :legal, nil].include?(only)
  return true
elsif identity_type.blank? && ValidatesIdentity.identity_type_aliases.keys.include?(only)
  return true
else
  false
end
```

WDYT?

Reply:
I agree with it, it makes it more clear.

Reply:
Commited